crates/tests/standalone/Cargo.toml

```diff
@@ -4,8 +4,16 @@ version = "0.0.0"
 edition = "2021"
 publish = false
 
+[lib]
+doc = false
+doctest = false
```

Comment:
Why is this needed?

Reply:
Ah, this was meant to be in a different PR.  I'll remove.